installer/README.md

```diff
@@ -458,6 +458,22 @@ Kubernetes objects, such as your TLS certificate or connection secrets.
 kubectl create namespace gitpod
 ```
 
+# I need to specify the load balancer - what do I do?
+
+By default, this will create a `proxy` service with `ServiceType: LoadBalancer`.
+If you need to specify your own load balancer, you can do this easily by changing
+your Installer configuration:
+
+```yaml
+meta:
+  ingress:
+    enabled: false
```

Comment:
We should make the behaviour of this flag more obvious in its naming, e.g. directly refer to the type of the proxy service.